### ISSUE TITLE
Fails to compile due to a dependency error

### ISSUE THREAD

**Issue by sdankel** [ourovoros-io/charcoal#4]
This just started failing in the last hour or so. I noticed it because it runs in CI on https://github.com/FuelLabs/sway-playground. It fails with the `--locked` flag as well.

`cargo install --git https://github.com/ourovoros-io/charcoal.git --rev 7c2fd7d1539d15304616269f11dbe20c1bd5d6ca`

```
error[E0023]: this pattern has 3 fields, but the corresponding tuple variant has 1 field
   --> src/project.rs:170:59
    |
170 |                 solidity::SourceUnitPart::PragmaDirective(_, _, _) => {
    |                                                           ^  ^  ^ expected 1 field, found 3
    |
   ::: /Users/sophiedankel/.cargo/registry/src/index.crates.io-6f17d22bba15001f/solang-parser-0.3.4/src/pt.rs:349:21
    |
349 |     PragmaDirective(Box<PragmaDirective>),
    |                     -------------------- tuple variant has 1 field

For more information about this error, try `rustc --explain E0023`.
```

**Comment by sdankel** [comment]
There was also a new crate version released of the broken dependency in the last hour: https://crates.io/crates/solang-parser

**Comment by sdankel** [comment]
It's weird that this broke since the dependencies in this project are pinned. It seems like the maintainers of the solana parser crate must have updated the code in the existing 0.3.3 release (by mistake presumably) when they released 0.3.4.
